tests/misc_env/install_prereq.py

```diff
@@ -134,11 +138,12 @@ def install_prereq(
 
 def setup_addition_repo(ceph, repo):
     log.info("Adding addition repo {repo} to {sn}".format(repo=repo, sn=ceph.shortname))
-    ceph.exec_command(
-        sudo=True,
-        cmd="curl -o /etc/yum.repos.d/rh_add_repo.repo {repo}".format(repo=repo),
+    base_repo = Ceph.generate_repository_file(repo, ["ceph-tools"])
+    base_file = ceph.remote_file(
+        sudo=True, file_name="/etc/yum.repos.d/rh_ceph.repo", file_mode="w"
     )
-    ceph.exec_command(sudo=True, cmd="yum update metadata", check_ec=False)
+    base_file.write(base_repo)
+    base_file.flush()
```

Comment:
IMO, we should change the existing implementation

Reply:
Done